Dockerfile

```diff
@@ -12,15 +12,16 @@ RUN go mod download
 # Copy the go source
 COPY . /workspace
 
-# Build with make to apply all build logic difined in Mekefile
+# Build with make to apply all build logic defined in Makefile
 RUN make build
+# Build host-local cni
+RUN git clone https://github.com/containernetworking/plugins.git ; cd plugins ; git checkout v1.2.0 -b v1.2.0
+RUN cd plugins ; go build -o plugins/bin/host-local ./plugins/ipam/host-local
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/base-debian11:latest
 WORKDIR /
 COPY --from=builder /workspace/build/ipam-controller .
 COPY --from=builder /workspace/build/ipam-node .
-USER 65532:65532
```

Comment:
the nonroot image and user cause  permission issue when copy the host-local cni to host